assets/package.json

```diff
@@ -75,11 +75,11 @@
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-uglify": "^6.0.3",
-    "uglify-js": "^3.6.0",
-    "uglifycss": "0.0.29",
```

Comment:
using cssnano rather tha uglifycss. This actually compresses better and is recommended by USWDS.